packages/data-broker/src/DataBroker/checkPermissions.ts

```diff
@@ -0,0 +1,115 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import { AccessPolicy } from '@tupaia/access-policy';
+import { DataBrokerModelRegistry, DataElement, DataSource } from '../types';
+import { fetchOrgUnitsByCountry } from './fetchOrgUnitsByCountry';
+
+const BES_ADMIN_PERMISSION_GROUP = 'BES Admin';
+
+const getUserPermissions = (accessPolicy?: AccessPolicy, countryCodes?: string[]) => {
```

Comment:
This diff is purely moving those functions from `DataBroker`, and changing the function signatures to accept all required input